release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,32 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+Pod admission is enforced by both pod security and security context constraint (SCC) admissions. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
+
+A controller monitors the xref:../authentication/managing-security-context-constraints.adoc[SCC-related permissions] of service accounts in user-created namespaces and automatically labels these namespaces with link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security admission] `warn` and `audit` labels.
+
+To improve workload security in accordance with the "restricted" pod security profile, this release introduces SCCs that enforce pod security in accordance with new pod security admission controls. These SCCs are:
+
+* `restricted-v2`
+* `hostnetwork-v2`
+* `nonroot-v2`
+
+They correspond to older, similarly named SCCs, but with the following enhancements:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
```

Comment:
IMO we need a more detailed DELTA between v1 and v2. On top of that. we might want to be more specific here. See:

> The `NET_BIND_SERVICE` capability can be added explicitly. 

For restricted-v2 we cannot add this one and the text give the impression that it is also valid for restricted-v2.

Suggestions:
a) Ensure that we clarify that:

- V2 does not permit allowPrivilegeEscalation=true (empty or false is compatible with v1 SCC and therefore works on previous OCP versions < 4.11)
- V2 requires you to either leave the dropped capabilities empty, or set it to ALL, or either allow a subset. Note that by being accepted as restricted-v2 the SCC will drop ALL always. V1 only dropped KILL,MKNOD,SETUID,SETGID capabilities while V2 drops ALL
- V2 requires you to either leave seccompProfile empty or set it to runtime/default (empty is compatible with v1 and works on previous OCP versions < 4.11 )

(ihmo) would be great if we could create a table that compares each policy from v1 to v2

Reply:
c/c @stlaz